### DISCUSSION TITLE
No me sale la kata 0, soy nuevo en esto. AYUDA POR FAVOR

### DISCUSSION THREAD

**Discussion by RupertoTacuche** [LaunchX-InnovaccionVirtual/CursoIntroPython#319] [Q&A] [answered]
Hola a todos, soy nuevo en python y no se como corregir esto que me sale de la kata 0
Anexo una imagen:

![image](https://user-images.githubusercontent.com/86775311/154829285-cb278725-62df-46ce-9549-c9bfd47a1644.png)

Alguien me puede ayudar?

**Comment by HectordAponte** [comment, marked as answer]
Hola, @RupertoTacuche 
Trataré de ayudarte. Entiendo que cuando sale la palomita es que la bilbioteca fue instalada correctamente. Los mensajes siguientes indican que ya la tienes instalada y estás repitiendo la instrucción.

![image](https://user-images.githubusercontent.com/99517955/154831467-6959e126-24ae-448b-8ff4-e9a313d15c7d.png)

Si tienes Windows, a mi la instrucción que me funcionó fue:
`!pip install ipywidgets` , con el símbolo de admiración al principio.

De cualquier forma, lo importante es que veas si puedes correr la siguiente celda de la notebook donde se usa la biblioteca. Si tienes más dudas te sugiero revisar las discusiones que postearon los que tuvieron problema con **ipywidgets**:

[Discusiones sobre ipywidgets](https://github.com/LaunchX-InnovaccionVirtual/CursoIntroPython/discussions?discussions_q=ipywidgets)

**Reply by RupertoTacuche** [reply]
Gracias, resolvi el problema

**Comment by LuceroLuciano** [comment]
Hola!
Puedes revisar si tienes la biblioteca `ipywidgets`  instalada,  con el siguiente comando: `pip list`, si no esta instalada, intenta instalarla nuevamente desde el cmd de Windows, yo la instale desde la terminal o consola cmd de Windows de la siguiente forma:   `pip install ipywidgets`

**Reply by RupertoTacuche** [reply]
Gracias
